Nighttime-Operations/Observatory/DIMM/ops.rst

```diff
@@ -0,0 +1,194 @@
+
+######################################
+DIMM Startup, Shutdown and Monitoring
+######################################
+
+The DIMM automatically performs seeing measurements from dusk until dawn. Limited support from the user is expected. 
+
+The vendor provided DIMM software will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.  
+At dawn, the telescope will park itself, and the dome will close. 
+This cycle repeats the following evening, unless :ref:`manually shutdown <Dimm-Shutdown>`, when the sun sets.
+
+The `ameba` program is continously checking that the environment conditions are stable and clear, based on the data delivered by `tt-meteo`.
+In case of adverse weather, `ameba` will close the dome until conditions clear. 
+
+Currently, there is no incoming Cerro Pachón weather information feeding the `tt-meteo` program. 
+
+Thus prior to starting up the DIMM, there are additional steps outlined in the :ref:`prerequisites section <Dimm_StartUp-Prerequisites>`.
+
+.. _Dimm_StartUp-Prerequisites: 
+
+Prerequisites: Weather Data
+===============================
+
+Every time the DIMM is restarted, the weather startup values are set so the operations are disabled for safety reasons. 
+
+To allow operations, mock weather data into the `tt-weather` service must be entered in the following manner:
+
+.. important::
+    Ensure the weather conditions are stable and safe before attempting the DIMM startup. 
+
+
+From a terminal in the LSST WAN network, connect to the DIMM VM:
+
+.. code-block:: bash
+
+        (base) you@Ios ~ % ssh dimm@dimm.cp.lsst.org
+
+The password is located on the Operators 1Password Vault.
+ 
+
+Connect via telnet to the ``tt-meteo``. 
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  telnet 127.0.0.1 16301
+
+
+The prompt within OpenTPL, a plain-text client-server protocol, is ``AUTH OK 0 0``. 
+Commands are prepended with a freely chosen command id (unsigned integer).
+More on this communication protocol can be found on the :ref:`DIMM general specs <DIMM-DIMM>`.
+
+The following two commands can be written once the prompt appears. 
+
+The first command will set the corrected sky temperature; the second will fix the sky to clear, confirming both observations are possible and safe. 
+
+.. code-block:: bash
+    
+        1 set sky.temp=-30
+    
+        2 set sky.status=0
+
+Quit telnet, pressing simultaneously ``Ctrl + ]``, write ``quit`` and click Enter. 
+
+The image below shows how the interaction with the OpenTPL client looks like and the two commands required to manually configure `tt-meteo`.
+
+.. figure:: ./_static/tt-meteo.png
+    :name: tt-meteo prerequisite
+
+    Prerequisite: Weather data
+
+Now the DIMM is ready to start operations. 
+
+Follow only one of the procedures below, either start the DIMM up in :ref:`automatic mode <Dimm_StartUp-Automatic>` or in 
+:ref:`manual mode <Dimm_StartUp-Manual>`
```

Comment:
this should be on line 78 with the rest of the sentence

Reply:
👍